cmd/system-probe/app/runtime-embed-tests.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package app
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/security/embeddedtests"
+	"github.com/spf13/cobra"
+)
+
+var (
+	runtimeEmbeddedTestsCmd = &cobra.Command{
+		Use:   "runtime-embedded-tests",
```

Comment:
I think something like `test-cws` might be clearer (and shorter! 😄 ). Runtime is implied by needing to run a command, and embedded is an implementation detail.

Reply:
Absolutely this is quite a mouthful ! Thanks for taking a look, this is still very much a PoC 😁